src/librustc_errors/emitter.rs

```diff
@@ -243,6 +243,7 @@ impl EmitterWriter {
                         end_col: hi.col_display,
                         is_primary: span_label.is_primary,
                         label: span_label.label.clone(),
+                        overlaps: false,
```

Comment:
```suggestion
                        overlaps_exactly: false,
```

I think this might be a clearer name.